src/components/PostDetails.tsx

```diff
@@ -132,7 +267,7 @@
                     type="button"
                     className="delete is-small"
                     aria-label="delete"
-                    onClick={() => deleteComment(comment.id)}
+                    onClick={() => handleDeleteComment(comment.id)}
```

Comment:
Ensure that the `handleDeleteComment` function properly handles errors. Currently, the catch block is empty, which means any errors during the deletion process will be silently ignored. Consider adding error handling logic to inform the user if a deletion fails.